descarteslabs/client/scripts/__main__.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
```

Comment:
I think a change is required in setup.py because the scripts moved to a different package